README.md

```diff
@@ -162,6 +162,14 @@ $ curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/aws-clo
 ```
 Remember to add `~/.guard/bin/` to your `$PATH`.
 
+Altenatively, you can install the latest version with [Homebrew](https://brew.sh/).
+
+```bash
+$ brew install cloudformation-guard
+```
+
+In this way, you need not to modify your `$PATH`.
```

Comment:
Can you please change this line to, 
"You would not need to modify \`$PATH\` this way."

Reply:
Changed. Thanks 🙇